app/src/main/java/com/example/android/mobilecourse/MoviesFragment.java

```diff
@@ -0,0 +1,121 @@
+package com.example.android.mobilecourse;
+
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import java.util.List;
+import java.util.Objects;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class MoviesFragment extends Fragment implements CustomAdapter.OnItemListener{
+
+    public static final String EXTRA_URL = "imageUrl";
+    public static final String EXTRA_TITTLE = "title";
+    public static final String EXTRA_DESCRIPTION = "description";
+    public static final String EXTRA_YEAR = "year";
+    private CustomAdapter adapter;
+    private RecyclerView recyclerView;
+    private SwipeRefreshLayout swipeRefreshLayout;
+    private LinearLayout linearLayout;
+    private View movieFragment;
+
+    public MoviesFragment() {}
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        movieFragment = inflater.inflate(R.layout.fragment_movies, container, false);
+
+        initViews();
+        loadMovies();
+        registerNetworkMonitoring();
+
+        return movieFragment;
+    }
+
+    private void initViews() {
+        recyclerView = movieFragment.findViewById(R.id.welcome_recycler_view);
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+        linearLayout = movieFragment.findViewById(R.id.movie_fr_linearLayout);
+        swipeRefreshLayout = movieFragment.findViewById(R.id.welcome_swipe_refresh);
+        setupSwipeToRefresh();
+    }
+
+    private void loadMovies(){
```

Comment:
форматування трохи поплило
```suggestion
    private void loadMovies() {
```